File: python-sdk/src/epilot/models/operations/getuser.py
import dataclasses
from typing import Optional
from ..shared import user as shared_user


@dataclasses.dataclass
class GetUserPathParams:
    id: str = dataclasses.field(metadata={'path_param': { 'field_name': 'id', 'style': 'simple', 'explode': False }})
    

@dataclasses.dataclass
class GetUserRequest:
    path_params: GetUserPathParams = dataclasses.field()
    

@dataclasses.dataclass
class GetUserResponse:
    content_type: str = dataclasses.field()
    status_code: int = dataclasses.field()
    user: Optional[shared_user.User] = dataclasses.field(default=None)
    

File: user-api/go-sdk/pkg/models/operations/getuser.go
package operations

import (
	"openapi/pkg/models/shared"
)

type GetUserPathParams struct {
	ID string `pathParam:"style=simple,explode=false,name=id"`
}

type GetUserRequest struct {
	PathParams GetUserPathParams
}

type GetUserResponse struct {
	ContentType string
	StatusCode  int64
	User        *shared.User
}
